empress_gui.py

```diff
@@ -4,364 +4,812 @@
+            input_file = tk.filedialog.askopenfilename(initialdir=os.getcwd(), title="Select a host file")
+            if input_file != "":
+                try:
+                    self.recon_input.read_host(input_file)
+                except Exception as e:
+                    messagebox.showinfo("Warning", "Error: " + str(e))              
+                self.host_file_path = input_file
+                # Force a sequence of loading host tree file first, and then parasite tree file, and then mapping file
+                self.load_files_dropdown['menu'].entryconfigure("Load parasite tree file", state = "disabled")
+                self.load_files_dropdown['menu'].entryconfigure("Load mapping file", state = "disabled")
+                self.refresh_when_new_input_files_loaded("Load host tree file")    
+                self.update_input_files_info("host")  
+
+        # Clicking on "Load parasite tree file" 
+        elif self.load_files_var.get() == "Load parasite tree file":
+            self.load_files_var.set("Load files")
+            # initialdir is set to be the current working directory
+            input_file = tk.filedialog.askopenfilename(initialdir=os.getcwd(), title="Select a parasite file")
+            if input_file != "":
+                try:
+                    self.recon_input.read_parasite(input_file)
+                except Exception as e:
+                    messagebox.showinfo("Warning", "Error: " + str(e))      
+                self.parasite_file_path = input_file
+                self.refresh_when_new_input_files_loaded("Load parasite tree file")
+                self.update_input_files_info("parasite")
+
+        # Clicking on "Load mapping file" 
+        elif self.load_files_var.get() == "Load mapping file":
+            self.load_files_var.set("Load files")
+            # initialdir is set to be the current working directory
+            input_file = tk.filedialog.askopenfilename(initialdir=os.getcwd(), title="Select a mapping file")
+            if input_file != "":
+                try:
+                    self.recon_input.read_mapping(input_file)
+                except Exception as e:
+                    messagebox.showinfo("Warning", "Error: " + str(e))       
+                self.mapping_file_path = input_file
+                self.refresh_when_new_input_files_loaded("Load mapping file")
+                self.update_input_files_info("mapping")
+                if self.recon_input.is_complete(): 
+                    self.when_recon_input_is_complete()            
+    
+    def when_recon_input_is_complete(self):
+        self.view_tanglegram_btn.configure(state=tk.NORMAL)
+        self.view_cost_space_btn.configure(state=tk.NORMAL)
+        self.compute_reconciliations_btn.configure(state=tk.NORMAL)
+        self.dtl_cost()
+        self.recon_input_wrapper = empress.ReconInputWrapper.from_files(self.host_file_path, self.parasite_file_path, self.mapping_file_path)
```

Comment:
Don't create a new class and use the `self.recon_input` which is now of type `empress.ReconInputWrapper`.